charts/kubernetes-external-secrets/values.yaml

```diff
@@ -125,3 +125,19 @@ serviceMonitor:
   enabled: false
   interval: "30s"
   namespace:
+
+deploymentInitContainers: {}
+
+# Add in additional named volumes and volume mounts to the deployment
+#
+#extraVolumes:
+#  namedVolume:
+#    emptyDir: {}
+#
+#extraVolumeMounts:
+#  namedVolume:
+#    mountPath: /usr/path
+#    readOnly: false
```

Comment:
```suggestion
extraVolumes: []
#  - name: namedVolume
#    emptyDir: {}
#
extraVolumeMounts: []
# - name: namedVolume
#    mountPath: /usr/path
#    readOnly: false
```

Reply:
All good - more likely me out of the loop ;) Updated.